virtcontainers/kata_agent.go

```diff
@@ -485,6 +485,10 @@ func (k *kataAgent) startSandbox(sandbox Sandbox) error {
 		}
 	}
 
+	if sandbox.config.HypervisorConfig.Msize9p != 0 {
+		sharedDir9pOptions = append(sharedDir9pOptions, fmt.Sprintf("msize=%d", sandbox.config.HypervisorConfig.Msize9p))
```

Comment:
I think this check, along with setting the default to 8192, is slightly confusing.
If I don't set an msize in the toml, then no msize is set on the mount line (as per above). The implication, in my mind, of the 'default=8192' is that we would set the msize to 8192 if it were not set in the toml. Now, I know that the *current* QEMU default is 8192 you say, so we have an implicit default being set to 8192, but if QEMU ever changed their default, then our code would be quite misleading...

I think the solution I see here is to always set the msize on the mount line - either to the toml value if set, or the default if not. What do you think @amshinde @devimc